### PR TITLE
Add about renaming existing packages

### LINKED ISSUE #7555: What are the steps to migrate a registered package to an organization?
I would like to migrate some package that are already registered in the registry to a GitHub organization I've just created. I couldn't find the steps in the README FAQ, could you please confirm that I can just proceed with the normal GitHub steps to migrate, and then later submit a PR here to update the URL?

### PR DESCRIPTION
**Pull request by oxinabox**
From discussion with @ChrisRackauckas and others on Slack

Fixes #7555

### CODE REVIEW

**Review by StefanKarpinski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -108,11 +108,35 @@ for users to navigate Julia's many varied packages.
...
-* Rename the GitHub repository to `NewName.jl`
+* [Rename the GitHub repository](github-rename) to `NewName.jl`
 * Rename the file `src/OldName.jl` to `src/NewName.jl`
 * Rename the top-level module to `NewName`
 * Update tests, documentation, etc, to reference the new name
 
+#### How do I rename an existing registered package?
+Technically, you can't rename a package once registered, as this would break existing users.
+But you can re-register the package again under a new name.
+Which has basically the same effecrt.
```
Comment (nickrobinson251) [inline README.md]: typo :)
```suggestion
Which has basically the same effect.
```

File: README.md [hunk 2]
```diff
@@ -108,11 +108,35 @@ for users to navigate Julia's many varied packages.
...
 
-* Rename the GitHub repository to `NewName.jl`
+* [Rename the GitHub repository](github-rename) to `NewName.jl`
 * Rename the file `src/OldName.jl` to `src/NewName.jl`
 * Rename the top-level module to `NewName`
 * Update tests, documentation, etc, to reference the new name
 
+#### How do I rename an existing registered package?
+Technically, you can't rename a package once registered, as this would break existing users.
+But you can re-register the package again under a new name.
```
Comment (nickrobinson251) [inline README.md]: maybe make clearer it has anew UUID too?
```suggestion
But you can re-register the package again under a new name with a new UUID.
```

File: README.md [hunk 3]
```diff
@@ -108,11 +108,35 @@ for users to navigate Julia's many varied packages.
...
+ - Follow the instructions above for renaminging a package: rename on GitHub, rename files etc.
+ - Generate a new UUID for the Project.toml
+ - Increment the version in the Project.toml as a breaking change.
+ - [Register](#registering-a-package-in-general) it as if it were a new package
+ - Comment on the PR, that this is a rename.
+ - It will have to go though the normal criteria for registring a new package.
+    - In particular, even if you get it merged manually, it will need to wait 3 days from the PR being openned.
+    - This gives others and youself the chance to point out any naming issues.
+
+You also should let your down users know about the rename, e.g. by placing a note in the README,
```
Comment (nickrobinson251) [inline README.md]: typo -- stray "down"
```suggestion
You also should let your users know about the rename, e.g. by placing a note in the README,
```

File: README.md [hunk 4]
```diff
@@ -108,11 +108,35 @@ for users to navigate Julia's many varied packages.
...
+or openning PRs/issues on downstream packages to change over.
```
Comment (nickrobinson251) [inline README.md]: typo 
```suggestion 
or opening PRs/issues on downstream packages to change over.
```

**Review by nickrobinson251 — COMMENTED** [review]
thanks for adding this 👍

File: README.md [hunk 5]
```diff
@@ -108,11 +108,35 @@ for users to navigate Julia's many varied packages.
...
+But you can re-register the package again under a new name with a new UUID.
+Which has basically the same effect.
+
+ - Follow the instructions above for renaminging a package: rename on GitHub, rename files etc.
+ - Generate a new UUID for the Project.toml
+ - Increment the version in the Project.toml as a breaking change.
+ - [Register](#registering-a-package-in-general) it as if it were a new package
+ - Comment on the PR, that this is a rename.
+ - It will have to go though the normal criteria for registring a new package.
+    - In particular, even if you get it merged manually, it will need to wait 3 days from the PR being openned.
```
Comment (ericphanson) [inline README.md]: Should be spelled “opened”

File: README.md [hunk 6]
```diff
@@ -108,11 +108,35 @@ for users to navigate Julia's many varied packages.
...
+    - This gives others and youself the chance to point out any naming issues.
```
Comment (ericphanson) [inline README.md]: Should be “yourself” instead of “youself”

**Review by ericphanson — COMMENTED** [review]
Looks very helpful!

File: README.md (hunk 6 above)
Comment (oxinabox) [reply README.md]:
```suggestion
    - This gives others and yourself the chance to point out any naming issues.
```

File: README.md (hunk 5 above)
Comment (oxinabox) [reply README.md]:
```suggestion
    - In particular, even if you get it merged manually, it will need to wait 3 days from the PR being opened.
```

**Review by DilumAluthge — APPROVED** [review]

**Comment by DilumAluthge** [comment]
It seems like there is a consensus in favor of this PR.

If nobody objects, I’ll merge this PR tomorrow.

**Pull request merged**